src/niclips/typing.py

```diff
@@ -1,13 +1,13 @@
 """Defined types used in niclips."""
 
 import os
-from typing import Tuple, Union
+from typing import Union
 
 import nibabel as nib
 import numpy as np
 
 StrPath = Union[str, os.PathLike]
 
-Coord = Tuple[float, float, float]
+Coord = Union[tuple[float, float, float] | np.ndarray]
```

Comment:
This looks odd, is it correct?

Reply:
Hmm, this is one place where it doesn't look like the formatter caught the updates. I'll go through these.